### PR TITLE
scaffold for cars

### PR DESCRIPTION
**Pull request by chitrak-bhatt**

### CODE REVIEW

File: app/models/car.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+class Car < ApplicationRecord
+  validates :company, :model, :purchase_date, 
+            :car_description, :registered_number, presence: true
+  validates :seats, :distance_driven, numericality: { only_integer: true }, presence: true       
+  enum engine_type: {
```
Comment (neshmapatel-simformsolutions) [inline app/models/car.rb]: you can simply define as;
enum engine_type: [:petrol, :diesel, :hybrid, :ev ]

File: app/models/car.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    Petrol: 0,
+    Diesel: 1,
+    Hybrid: 2,
+    EV: 3
+  }
+
+  enum car_type: {
+    Hatchback: 0,
```
Comment (neshmapatel-simformsolutions) [inline app/models/car.rb]: same as above

File: app/models/car.rb [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    Sedan: 1,
+    SUV: 2,
+    Supercar: 3
+  }
+
+  enum transmission_type: {
+    Manual: 0,
```
Comment (neshmapatel-simformsolutions) [inline app/models/car.rb]: here as well.

**Review by neshmapatel-simformsolutions — CHANGES_REQUESTED** [review]

File: app/models/car.rb (hunk 2 above)
Comment (chitrak-bhatt) [reply app/models/car.rb]: Done

File: app/assets/stylesheets/custom.css [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    line-height: 1.5;
+    color: #212529;
+    background-color: #fff;
+    border: 1px solid #ced4da;
+    border-radius: 0.25rem;
+    transition: border-color .15s ease-in-out, box-shadow .15s ease-in-out;
+    -webkit-appearance: none;
+    -moz-appearance: none;
+    appearance: none;
+}
```
Comment (neshmapatel-simformsolutions) [inline app/assets/stylesheets/custom.css]: add a new line at EOF.

Comment (chitrak-bhatt) [reply app/assets/stylesheets/custom.css]: Done

**Review by neshmapatel-simformsolutions — APPROVED** [review]

**Pull request merged**